src/components/Error/Error.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+
+import { TypeError } from '../../types/TypeError';
+
+type Props = {
+  errorMessage: TypeError;
+  setErrorMessage: React.Dispatch<React.SetStateAction<TypeError>>;
+};
+
+export const Error: React.FC<Props> = ({ errorMessage, setErrorMessage }) => {
+  useEffect(() => {
+    if (errorMessage) {
+      setTimeout(() => setErrorMessage(TypeError.DEFAULT), 3000);
+    }
+  }, [errorMessage]);
```

Comment:
Add setErrorMessage as a dependency

React Hook useEffect has a missing dependency: 'setErrorMessage'. Either include it or remove the dependency array. If 'setErrorMessage' changes too often, find the parent component that defines it and wrap that definition in useCallback